evaluation.md

```diff
@@ -5,6 +5,31 @@
 _Describe each change from your ideal syntax to the syntax you implemented, and
 describe_ why _you made the change._
 
+First, we added a package line and put all the code in an object, because we are
+pretty sure they are mandatory. We got rid of the curly braces around the state
+declarations because we realized we could go without them and decided they were
+not the most beautiful. We added "and" between the direction and the next state
+because the parser appears to require something like "object, method, object,
+method...". We added pre-declarations of the states because it was important to
```

Comment:
This was my biggest pet peeve, and the largest indication (to me) that we were very much pushing the bounds of an internal DSL.  And by pushing I mean shattering and stomping all over its grave.